src/erlang_ls.erl

```diff
@@ -64,11 +68,12 @@ lager_config() ->
 
 -spec lager_handlers(string()) -> [any()].
 lager_handlers(LogRoot) ->
+  LoggingLevel = application:get_env(?APP, log_level, ?DEFAULT_LOGGING_LEVEL),
   LogFile = filename:join([LogRoot, "debug.log"]),
```

Comment:
Shouldn't the filename also depend on the level now? Sorry I didn't catch this earlier.

Reply:
Might make sense to keep it all in one place and just rename the file. Maybe just rename it to `server.log`?

Reply:
IMO keeping the name of the logging level makes it less surprising since this is in general what `lager` does.

Reply:
I think one problem with the logging level as a file name is that it may cause issues on Window machines, as far as I know we can't have `<` or `>` in file names which might occur with the way we're setting this up.

Reply:
Made the suggested change. Made the following mappings in the filename to avoid the above:

```
">" -> "gt"
"<" -> "lt"
"=" -> "eq"
"!" -> "not"
```

Wasn't necessary to switch "=" and "!" but felt it made more sense to do so.

Reply:
I didn't realize before the idea was to support the full lager [syslog style comparison flags](https://github.com/erlang-lager/lager#syslog-style-loglevel-comparison-flags). In that case I would definitely go for just using `server.log`. Sorry for generating the extra work 😞 🙏 .

Reply:
TIL. I didn't realize that either O_o

Reply:
Haha, yeah that was the general idea since we're directly exposing lager's log level. Any reason not to?

Reply:
After learning about the syslog style comparison flags, it feels like @benrs original idea of having a simple `server.log` is actually the way to go. Also, we should update the README to reflect these changes. I will merge this PR as it is and PR a follow-up.